src/client/java/aztech/modern_industrialization/compat/waila/client/PipeComponentProvider.java

```diff
@@ -168,28 +179,4 @@ public void appendBody(ITooltip tooltip, IBlockAccessor accessor, IPluginConfig
         }
     }
 
-    /**
-     * Try to find a better color if possible by reflecting into megane's internals ;-)
```

Comment:
Is there anything like this? Otherwise IIRC Lava is shown in gray instead of being shown in orange

Reply:
Maybe this is a problem that is better fixed with the transfer API... 🤔

Reply:
WTHIT's fluid bar actually render the fluid texture, it uses the transfer api method.

Reply:
Ohhhh,that's great. ;)